src/main/java/com/javadiscord/jdi/internal/api/DiscordRequestBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.javadiscord.jdi.internal.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DiscordRequestBuilder {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private String method;
```

Comment:
default to `GET`?

Reply:
Let's see how it pans out and leave this as future tech debt